packages/client/hmi-client/src/components/widgets/tera-input.vue

```diff
@@ -42,6 +43,10 @@ const error = ref('');
 const maskedValue = ref('');
 
 const isNistType = props.type === 'nist';
+// If the input is a string composed only of digits, display as NIST number
+const isTextContainingOnlyDigits =
+	isString(props.modelValue) && /^\d+(\.\d+)?$/.test(props.modelValue);
```

Comment:
should this be a `computed` if the input turns into a mix of digits and letters and then changes to just digits?

Reply:
Fair point, I think my latest changes mess that up, will update that.